mne/viz/_3d.py

```diff
@@ -1344,3 +1346,160 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def plot_dipole_mri_orthoview(dipole, trans, subject, subjects_dir=None,
```

Comment:
OK I see that I can use interactions to see all dipoles but I am thinking this is a surprising as a default. I would expect to see all dipoles by default. Can we have an interactive=True | False param?